Libraries/LibGUI/ProcessChooser.h

```diff
@@ -46,6 +53,10 @@ class ProcessChooser final : public GUI::Dialog {
     String m_window_title;
     String m_button_label;
     RefPtr<Gfx::Bitmap> m_window_icon;
+
+    bool m_refresh_enabled { true };
+    unsigned int m_refresh_interval { 150 };
```

Comment:
This is very aggressive, let's go with 1000 ms instead.
Also, `unsigned int` -> `unsigned`.

Reply:
hint hint `u32` looks better hint hint